### PR TITLE
Add Docs For Latest Security Patch Image Tag

### PR DESCRIPTION
**Pull request by rasa-jmac**
**Proposed changes**:
- Add docs line to explain the addition of a new `[release]-latest` tag to the security patched images for Rasa Plus.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Review by miraai — APPROVED** [review]
Thank you!

File: docs/docs/deploy/deploy-rasa.mdx [hunk 1]
```diff
@@ -202,7 +202,7 @@ and [specify a pull secret for the image](#step-1-image-pull-secret).
 
 #### Security Patch Releases
 Beginning with Rasa Plus 3.4.0, we release a new Docker image for supported versions each day at 0800 UTC. These images have OS security patches applied and include the date of generation in their tag in the format `YYYYMMDD`. 
-For example, to use the 3.4.2 image generated on the 1st of February 2023, use the tag `3.4.2-20230201`. These images can optionally be used as a drop-in replacement for the same version of Rasa Plus, but with all OS security updates up to that date applied.
+For example, to use the 3.4.2 image generated on the 1st of February 2023, use the tag `3.4.2-20230201`. Alternatively, you can also ensure that you always fetch the most recent security patch release using the tag format `[release]-latest` eg `3.4.2-latest`. These images can optionally be used as a drop-in replacement for the same version of Rasa Plus, but with all OS security updates up to that date applied.
```
Comment (m-vdb) [inline docs/docs/deploy/deploy-rasa.mdx:205]: awesome 🙏🏻  Can I also ask you to add a link to this section in [the new Rasa Pro page](https://github.com/RasaHQ/rasa/blob/dd139604071715df6d8d04f434a975412f62dfac/docs/docs/rasa-pro.mdx?plain=1#L55)? At the bottom there's a section about "Security Scanning for Vulnerability Protection", and you'd just need to add:
```
(new line)
[Read more here](./deploy/deploy-rasa.mdx#security-patch-release).
```
(the link should be accurate but if it's not, it'll be caught by lint checks)

**Review by m-vdb — APPROVED** [review]
thanks!

File: docs/docs/deploy/deploy-rasa.mdx (hunk 1 above)
Comment (rasa-jmac) [reply docs/docs/deploy/deploy-rasa.mdx:205]: Good catch, I've added that now

**Pull request closed without merging**